jekyll/_cci2/executor-types.md

```diff
@@ -100,6 +100,30 @@ Docker Images may be specified in three ways, by the image name and version tag
 
 Nearly all of the public images on Docker Hub and Docker Registry are supported by default when you specify the `docker:` key in your `config.yml` file. If you want to work with private images/registries, please refer to [Using Private Images]({{ site.baseurl }}/2.0/private-images).
 
+### RAM disks
+
+If you suspect your builds are I/O limited, it can be helpful to use the default Docker RAM disk at `/dev/shm`. For certain builds, this can reduce build times dramatically.
```

Comment:
Should we be mentioning that it is helpful to use `/dev/shm` but then in the next sentence also say we cannot use it because it's `NOEXEC`?  

I would maybe write something to the effect that "A RAM disk is available at `/mnt/ramdisk` that offers a [temporary file storage paradigm](https://en.wikipedia.org/wiki/Tmpfs) similar to using `/dev/shm`.

Reply:
Done!